server/routes/caseNotes/caseNotesController.ts

```diff
@@ -14,7 +22,26 @@ export default class CaseNotesController {
       size: 5,
     }
     const caseNotesPage = await this.interventionsService.getCaseNotes(accessToken, req.params.id, paginationQuery)
-    const presenter = new CaseNotesPresenter(caseNotesPage)
+    const uniqueUsers = caseNotesPage.content
+      .map(caseNote => caseNote.sentBy)
+      // filter out duplicate username
+      .filter((user, index, self) => index === self.findIndex(otherUser => otherUser.username === user.username))
+    const userDetails: { username: string; officer: undefined | DeliusUser | AuthUserDetails }[] = await Promise.all(
+      uniqueUsers.map(async user => {
+        let officer: undefined | DeliusUser | AuthUserDetails
+        try {
+          if (user.authSource === 'delius') {
```

Comment:
you can do this using only hmpps auth - which is much preferred over CAPI. 

for an example see HMPPSAuthService::getUserDetail in the interventions service repo.

Reply:
oh is it more like a map of username -> display name? 

in that case why not make it a map?

`const displayNames: Map<string, string> = await Promise.all( .... )`

Reply:
I've added a new end point for hmpps auth '/api/user/$USERNAME'. This is a new commit.